src/components/biggive-page-section/biggive-page-section.stories.tsx

```diff
@@ -1,36 +1,29 @@
 import { colorSelector } from '../../color-selector';
+import { sectionStyles } from './biggive-page-section-prop-types';
 
 export default {
   title: 'Components/Layout',
   argTypes: {
     spaceBelow: {
       name: 'Space below',
```

Comment:
think it was just a mistake to have a 'control' key inside these and the stuff inside was always ignored until now.